### DISCUSSION TITLE
Showing entries only if they belong to a specific taxonomy

### DISCUSSION THREAD

**Discussion by tomasg-lt** [statamic/cms#5916] [Q&A] [answered]
I'm running a query:

`{{ collection:products paginate="15" as="posts" }} 

{{ /collection:products }}`

I want only to show products which belong to any of the taxonomy NEW ITEMS terms:

- CPU
- GPU
- MEMORY

etc. 

Is there a way to append query with taxonomy:new_items=ANY or something similar, to only show products that belong to at least one of the specified taxonomy terms?

Thanks!

**Comment by jasonvarga** [comment, marked as answer]
`taxonomy:new_items:any="cpu|gpu|memory"`

**Reply by tomasg-lt** [reply]
Thanks, I'm aware of that. Isn't there a way to use **new_items** and **:any** without listing the actual terms? So even client adds a new term in new_items taxonomy, there would be no need to update template.

**Reply by jasonvarga** [reply]
I see. Maybe you can just show entries that have _something_ selected.

```
{{ collection:products new_items:exists="true" paginate="15" as="posts" }}
```

**Reply by tomasg-lt** [reply]
Thanks, that's exactly what I needed. Most things in Statamic can be done easier than I thought they should be!

**Reply by tobinski** [reply]
This does not work anymore. The problem is, that after I save/update an entry statamic saves `new_items: []`. So even if i don't select any term, the field exists
